.travis.yml

```diff
@@ -20,27 +20,51 @@ before_install:
   - conda update -q conda
   # Useful for debugging any issues with conda
   - conda info -a
+
+  # install MHC predictors
+  - git clone https://mhcbundle:$MHC_BUNDLE_PASS@github.com/openvax/netmhc-bundle.git
+  - export NETMHC_BUNDLE_HOME=$PWD/netmhc-bundle
+  - mkdir tmp
+  - export NETMHC_BUNDLE_TMPDIR=$PWD/tmp
+  - export PATH=$PATH:$NETMHC_BUNDLE_HOME/bin
+
+  # needed for MHCflurry predictor test
+  - export KERAS_BACKEND=theano
+  # we want to include vaxrank unit tests as a general integration test to make sure the stack
+  # isn't broken in an obvious way
+  - git clone https://github.com/openvax/vaxrank.git
 addons:
   apt:
     packages:
-    # install pandoc for use with pypandoc for converting the README
-    # from markdown to RST
-    - pandoc
+      # Needed for NetMHC
+      - tcsh
+      # install pandoc for use with pypandoc for converting the README
+      # from markdown to RST
+      - pandoc
+env:
+  global:
+    # MHC_BUNDLE_PASS
+    - secure: "B2bLOYhDD1mSH6nBmUVoyoVzO8CsZ/qeduoQ/gq1UqPXyBa5cj8x1hYeYEXMVGH04dq6p5FHKIZoZOOmgtKoemje5enSV8IL2lttASqNvzsiyvy6I4C3r5Ve2H9HMLneDxkarfVVVhG2ZJyO7Y/vGN8STZWX6N3FJfCJ1B5kV5Q="
 cache:
-  pip: true
   # cache directory used for Ensembl downloads of GTF and FASTA files
   # along with the indexed db of intervals and ID mappings and pickles
   # of sequence dictionaries
-  directories: /home/travis/.cache/pyensembl/
+  directories:
+      - /home/travis/.cache/pyensembl/
+      - /home/travis/.cache/pip
 install:
   - >
       conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION
-      numpy nose pandas
+      numpy nose pandas theano pandoc
   - source activate test-environment
   - pip install pypandoc
   - pip install -r requirements.txt
   - pip install .
   - pip install coveralls
+  - pip install --upgrade -r $PWD/vaxrank/requirements.txt
+before_script:
+  # needed for MHCflurry predictor test
+  - mhcflurry-downloads fetch
```

Comment:
I think you can remove all `mhcflury` specific stuffs since `vaxrank` doesn't test it?

Reply:
done